maven/versions-rules.xml

```diff
@@ -2,18 +2,22 @@
 <!-- See https://stackoverflow.com/a/48715568 for details about this file -->
 <ruleset xmlns="http://mojo.codehaus.org/versions-maven-plugin/rule/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" comparisonMethod="maven" xsi:schemaLocation="http://mojo.codehaus.org/versions-maven-plugin/rule/2.0.0 http://mojo.codehaus.org/versions-maven-plugin/xsd/rule-2.0.0.xsd">
   <ignoreVersions>
-    <!-- Ignore Alpha's, Beta's, release candidates and milestones -->
-    <ignoreVersion type="regex">(?i).*Alpha(?:-?\d+)?</ignoreVersion>
-    <ignoreVersion type="regex">(?i).*a(?:-?\d+)?</ignoreVersion>
-    <ignoreVersion type="regex">(?i).*Beta(?:-?\d+)?</ignoreVersion>
-    <ignoreVersion type="regex">(?i).*-B(?:-?[\d\.]+)?</ignoreVersion>
-    <ignoreVersion type="regex">(?i).*-eap(?:-?\d+)?</ignoreVersion>
-    <ignoreVersion type="regex">(?i).*RC(?:-?\d+)?</ignoreVersion>
-    <ignoreVersion type="regex">(?i).*CR(?:-?\d+)?</ignoreVersion>
-    <ignoreVersion type="regex">(?i).*M(?:-?\d+)?</ignoreVersion>
-    <!-- Ignore .prX from jackson etc. -->
-    <ignoreVersion type="regex">(?i).*pr\d+</ignoreVersion>
-    <!-- Ignore org.postgresql:postgresql:42.2.5.jre7 and alike -->
-    <ignoreVersion type="regex">(?i).*jre\d+</ignoreVersion>
+    <!-- Ignore Alpha's, Beta's, release candidates, milestones and snapshots -->
+    <ignoreVersion type="regex">(?i).*alpha(-?\d+)?(.*)</ignoreVersion>
```

Comment:
Hvorfor har du lagt på `(.*)` ? Det betyr jo i praksis at den ledende `(-?\d+)?` ikke lenger har noen effekt

Reply:
Sistnevnte er lagt på for da de ikke matchet 
```
0.20.0-1.4-M1-release-99
0.20.0-1.3.70-eap-274-2
2.1.0-alpha.0.24+0fdeb6e
```

Reply:
Bør nok inn i den foregående gruppen på et vis.

Reply:
Kan man egentlig bare forenkle til følgende? 
```
(?i).*alpha(.*)
```